cmd/agent/main_windows.go

```diff
@@ -20,6 +21,20 @@ import (
 	"golang.org/x/sys/windows/svc"
 )
 
+// https://docs.microsoft.com/en-us/windows/console/handlerroutine
+const (
+	ctrlCEvent        = uint(0)
+	ctrlBreakEvent    = uint(1)
+	ctrlCloseEvent    = uint(2)
+	ctrlLogOffEvent   = uint(5)
+	ctrlShutdownEvent = uint(6)
+)
+
+var (
+	kernel32              = syscall.NewLazyDLL("kernel32.dll")
```

Comment:
I think it will prevent us from using Nano, right?

Reply:
Not sure if this is updated but it appears `SetConsoleCtrlHandler` is available under [api-ms-win-core-console-l1-1-0.dll](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/mt588480(v=vs.85)#api-ms-win-core-console-l1-1-0dll) on Nano Server if `kernel32.dll` is not an option.